src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -317,6 +377,10 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
             isLoadingClassification,
         }: State = this.state;
 
+        if (!file) {
+            return null; // TODO: change to loading indicator and handle errors once file call split out
```

Comment:
You can also look at `<LoadingIndicatorWrapper>` (used in MetadataSidebar.js) if that helps at all...though I think that requires some content to overlay...